### DISCUSSION TITLE
How can I configure Codecov badge color threshold?

### DISCUSSION THREAD

**Discussion by jlandowner** [badges/shields#10133] [Q&A] [answered]
Hi I am using Codecov badge in my project. My project badge color shows yellow.
[![codecov](https://codecov.io/gh/jlandowner/helm-chartsnap/graph/badge.svg?token=8ARG53CRL7)](https://codecov.io/gh/jlandowner/helm-chartsnap)
https://github.com/jlandowner/helm-chartsnap

Recently I found the other project using Codecov badge and also found that the coverage is lower than my prooject but it is green color.

[![codecov](https://codecov.io/gh/coder/code-server/branch/main/graph/badge.svg?token=5iM9farjnC)](https://codecov.io/gh/coder/code-server)
https://github.com/coder/code-server

I cannot found the difference in the URL.
How can I configure a coverage color threshold?

**Comment by chris48s** [comment, marked as answer]
The 2 READMEs you've linked there are both using coverage badges served from codecov.io:
https://codecov.io/gh/jlandowner/helm-chartsnap/graph/badge.svg?token=8ARG53CRL7
https://codecov.io/gh/coder/code-server/branch/main/graph/badge.svg?token=5iM9farjnC
They are not shields.io badges.

Consider contacting codecov support or raising an issue somewhere under https://github.com/codecov/ with your question.

**Reply by jlandowner** [reply]
Oh, sorry😓 Thanks you for your kind answer! I will contact codecov or try shield.io badge.
